python/cudf/cudf/core/dataframe.py

```diff
@@ -269,6 +271,21 @@ def __dir__(self):
         )
         return list(o)
 
+    def __setattr__(self, key, col):
+        try:
+            object.__getattribute__(self, key)
+            return object.__setattr__(self, key, col)
+        except AttributeError:
+            pass
+
+        try:
+            if self._enable_setattr_setitem:
+                self.__setitem__(key, col)
+        except AttributeError:
```

Comment:
I had a warning here. I'm not sure what happened to it. I'll get it back in there.